src/main/java/mousio/etcd4j/transport/EtcdNettyClient.java

```diff
@@ -392,7 +413,6 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws E
   @Override
   public void close() {
     logger.info("Shutting down Etcd4j Netty client");
-    this.timer.stop();
```

Comment:
should have a max delay, for example 5 seconds as default, instead of a infinite delay